fastmri/pl_modules/data_module.py

```diff
@@ -21,14 +21,40 @@ def worker_init_fn(worker_id):
     data: Union[
         SliceDataset, CombinedSliceDataset
     ] = worker_info.dataset  # pylint: disable=no-member
+
+    # Check if we are using DDP
+    is_ddp = False
+    if torch.distributed.is_available():
+        if torch.distributed.is_initialized():
+            is_ddp = True
+
     # for NumPy random seed we need it to be in this range
-    seed = worker_info.seed % (2 ** 32 - 1)  # pylint: disable=no-member
+    base_seed = worker_info.seed % (2 ** 32 - 1)  # pylint: disable=no-member
```

Comment:
Might be better to just do `% (2 ** 32 - 1)` at the `seed` function call.

Reply:
Sounds good. Do you mean only doing `% (2 ** 32 - 1)` at the very end at `dataset.transform.mask_func.rng.seed()` ?

Reply:
Yeah, it can be `dataset.transform.mask_func.rng.seed(seed % (2 ** 32 - 1))`, for example.